File: Galanin_Pavel/lab9/sources/app/src/main/java/com/example/gpi_rpodms6_lab9/MainActivity.java
package com.example.gpi_rpodms6_lab9;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void toVideo(View view) {
        Intent intent = new Intent(this, VideoActivity.class);
        startActivity(intent);
    }

    public void toAudio(View view) {
        Intent intent = new Intent(this, AudioActivity.class);
        startActivity(intent);
    }

    public void toCamera(View view) {
        Intent intent = new Intent(this, CameraActivity.class);
        startActivity(intent);
    }

    public void toGallery(View view) {
        Intent intent = new Intent(this, GalleryActivity.class);
        startActivity(intent);
    }
}

File: Kovalchuk_Vladislav/lab5/sources/app/src/main/java/by/brstu/redlabrat/testlistapp/MainActivity.kt
package by.brstu.redlabrat.testlistapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import by.brstu.redlabrat.testlistapp.api.OmdbSearchResultMovie

class MainActivity : AppCompatActivity(), MyListAdapter.MyItemClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    override fun onResume() {
        super.onResume()
        val fragment = ListFragment.newInstance("")
        supportFragmentManager.beginTransaction()
            .add(R.id.mainFrame, fragment)
            .commit()
    }

    override fun onMyItemClick(selectedMovie: OmdbSearchResultMovie) {
        supportFragmentManager.beginTransaction()
            .replace(R.id.mainFrame, DetailsFragment.newInstance(selectedMovie.title))
            .addToBackStack(null)
            .commit()
    }
}